### ISSUE TITLE
Finding a connection with the PublicCode.Directory

### ISSUE THREAD

**Issue by bvhme** [italia/publiccode.yml#38]
Dear all,

The great FSFE and OpenUK people have also come up with a _decentralised_ Public Code dissemination strategy at FOSDEM last year, what are the odds!

They've developed the [publiccode.directory](https://publiccode.directory) as a way for Public Coders to find Public Code projects. As part of the project they've also developed a JSON based datamodel for projects that aims to achieve very similar things as this standard.

They've expressed intent to figure out how we can combine forces with the two projects and combine our common goal of ultimate Public Code discoverability.

Check out the entries in their repository: https://github.com/OpenUK/publiccode.directory

**Comment by alranel** [comment]
Hi @bvhme, yes, let's do it!
Maybe we can talk about it under the new OSI wg umbrella.

**Comment by bvhme** [comment]
Yes, I’ve had a short chat with them and Italo from OSI, perhaps we can
plan a call sometime next week and tell each other about how we might be
able to combine these projects together.

**Comment by StuartJMackintosh** [comment]
Boris - give me a shout if you would like to pick up the conversation.

**Comment by StuartJMackintosh** [comment]
Thanks for the invitation - I can make later in the day on 11th - say after 16:00 CET, or Friday 12th any time.

**Comment by bvhme** [comment]
He, I think it would be great if you are there so I've updated the event, I
hope others can make it.

**Comment by bfabio** [comment]
I think we can close this, feel free to reopen it.
